### PR TITLE
update link, add explanation

### PR DESCRIPTION
**Pull request by gluap**
This updates the link to point to the new strengthened mounting bracket and adds to the description of printing materials.

@boldt asked for this on the forums.

I also took the time  to summarize the discussion from [the forum](https://forum.openbikesensor.org/t/hinweise-zum-3d-druck-der-sliderplatte-am-gehaeuse/121/8) about the strenghtened bracket and how it could be printed safely.

### CODE REVIEW

File: content/docs/hardware/general/case/_index.de.md [hunk 1]
```diff
@@ -170,9 +181,20 @@ liegen auf der [ADFC page - Vorlagen zum OpenBikeSensor ](https://aktiv.adfc.de/
...
+  Infill (>50%) zu drucken, und die Wandstärke auf 2.4mm zu erhöhen, 
+  respektive je nach Slicer bei einer 0.4mm Düse 6 Perimeter zu wählen. Das
+  Teil sollte seitlich stehend gedruckt werden, um die Extrusionsrichtung
+  in Belastungsrichtung auszurichten, was das Teil zusätzlich verstärkt. Bei
+  Haftungsproblemen kann das Teil in den Slicereinstellungen mit einem Brim
+  versehen werden.
+  {{< /imgproc >}}
+
+
+{{< imgproc "case parts - prusaslicer.png" Resize 800x >}}
```
Comment (SubOptimal) [inline content/docs/hardware/general/case/_index.de.md:197]: This image seems to be missing.

Comment (gluap) [reply content/docs/hardware/general/case/_index.de.md:197]: It's part of the pr (bottom most file in list of Files changed)

Comment (SubOptimal) [reply content/docs/hardware/general/case/_index.de.md:197]: User error on my side.

**Comment by opatut** [comment]
Do we still need this @gluap ?

**Comment by gluap** [comment]
Nope, already added to the openscad bracket.

**Pull request closed without merging**